documents/operating.md

```diff
@@ -13,10 +13,44 @@ Contents:
 ---------------------------------------------------------------------
 SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
-- [ ] Company term
-- [ ] Continuance of company
-- [ ] Rickroll
-- [ ] Game of Mario Party 3 before legal arbitration between members
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the board fails to
+hold a single meeting with quorum for a period of (a consecutive year?),
+(3) everyone dies / there are no more Corporate Overlords, or (4) if all
+Corporate Overlords are straight white cis men.
```

Comment:
100% agree on the first point. I caught that as well in my blind read-through.

Didn't think of the second point, but also totally agree that it should be explicitly stated.